library/build.gradle

```diff
@@ -15,6 +15,7 @@ android {
         aarMetadata {
             minCompileSdk = rootProject.ext.compileSdkVersion
         }
+        multiDexEnabled true
```

Comment:
This shouldn't be needed.

Reply:
Hm, I looked here. Do you know if it is wrong?
![image](https://user-images.githubusercontent.com/46191832/226713302-fe67f7dc-23ac-4722-922b-a2875841b195.png)

Reply:
Ah I see! I forgot this project was using minSdk 14. It should probably be bumped to 21 😅